### DISCUSSION TITLE
Scoring system in Task 2b

### DISCUSSION THREAD

**Discussion by TomohiroOno33** [hsr-project/general#11] [Q&A] [answered]
I think the scoring system for Task 2b is different from the rulebook.

In the rulebook, it is as follows:

> 1. Grasping an object from the shelf (100 points total)
> Taking any food item in the shelf: 40 points
> Taking the requested object among many objects in the shelf: +60 points
> 
> 2. Delivering the object to a person in the shelf (100 points total)
> Delivering the object to a person in the delivery area: 40 points
> Correctly detecting a person's request and giving the item to her/him: +60 points

However, in the scoring system, the scores are as follows:

> 1. Delivering the object to a left person (100 points total)
> Delivering the object to a left person in the delivery area: 40 points
> Delivered object is correct: +60 points
> 
> 2. Delivering the object to a right person (100 points total)
> Delivering the object to a right person in the delivery area: 40 points
> Delivered object is correct: +60 points

In other words, in the rulebook, 200 points are earned by correctly taking the specified object and delivering it to the specified person (left or right).
In the scoring system, however, this must be done twice correctly to get 200 points.

**Comment by yosuke** [comment]
That may be true.
But, it may be better to talk with the rule creators about this rule, because the scores of task2b are already considered too high and may break the game balance.
Also, detecting the correct grasp of an object by the automatic scoring system is actually very difficult.
I will come back after discussing with the rule creators.

**Comment by yosuke** [comment, marked as answer]
@TomohiroOno33 
I have talked to the rule creators and they have referred to this note in the rulebook:

```The rules and regulations of the sDSPL are highly based on the World Robot Summit - Partner Robot Challenge (Real Space), as in https://worldrobotsummit.org, and therefore some rules might not be applicable to a simulated environment, especially those limited by real physical and mechanical principles.```

Which means the first 100pt rule is not applied to the sDSPL at this time.

**Reply by TomohiroOno33** [reply]
Thank you for your reply.

What does the first 100 pts rule mean?
I think "Grasping an object from the shelf", is it correct?

**Reply by yosuke** [reply]
Sorry about the clarification.
Total 100pt given by the "grasping an object from the shelf" rule will be skipped in this specific sDSPL competition.
Only the "delivering the object to a person in the shelf" rule will be applied.

**Comment by yosuke** [comment]
@TomohiroOno33 You may have heard about this from a different channel, but a rule creator has pointed out that the delivery attempt should be only done once (not twice to each left and right person). So the score will be 100pt + bonus.
Sorry for my initial information.

**Reply by TomohiroOno33** [reply]
Yes, I understand.
Thank you.
